templates/vanilla/package.json

```diff
@@ -3,14 +3,17 @@
   "private": true,
   "scripts": {
     "build": "pnpm recursive run build",
-    "dev": "run-pty % pnpm dev:node % pnpm dev:client % pnpm dev:contracts",
+    "dev": "pnpm recursive run dev",
     "dev:client": "pnpm --filter 'client' run dev",
     "dev:contracts": "pnpm --filter 'contracts' dev",
-    "dev:node": "pnpm --filter 'contracts' devnode",
+    "dev:node": "mud devnode",
+    "dev:pty": "run-pty % pnpm dev:node % pnpm dev:client % pnpm dev:contracts",
     "initialize": "pnpm recursive run initialize",
+    "mud:up": "pnpm recursive exec mud set-version -v canary && pnpm install",
     "test": "pnpm recursive run test"
   },
   "devDependencies": {
+    "@latticexyz/cli": "link:../../packages/cli",
```

Comment:
@holic just double checking, will this also get replaced with the correct version when installing the template?

Reply:
I think so! https://github.com/latticexyz/mud/blob/main/packages/create-mud/scripts/copy-templates.sh

You can test by running `pnpm build` inside of `create-mud` and look at `dist/templates`

Reply:
yep that worked!